firstvoices/backend/views/search/base_search_views.py

```diff
@@ -121,6 +122,54 @@
                     ),
                 ],
             ),
+            OpenApiParameter(
+                name="kids",
+                description="Return kids-friendly entries only if true.",
```

Comment:
Minor suggested edit to avoid ambiguity: "Return only kids-friendly entries if true"